test/testSetup.ts

```diff
@@ -11,34 +11,37 @@ import { DefaultTreeItem, InputBox, QuickOpenBox } from 'wdio-vscode-service';
 import { EnvironmentSettings as Env } from './environmentSettings.ts';
 import * as utilities from './utilities/index.ts';
 import { fileURLToPath } from 'url';
+import { gitClone } from './utilities/gitCommands.ts';
 
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 
 export class TestSetup {
   public testSuiteSuffixName: string;
   private static aliasAndUserNameWereVerified = false;
-  public tempFolderPath: string | undefined = undefined;
+  public tempFolderPath: string;
   public projectFolderPath: string | undefined;
   private prompt: QuickOpenBox | InputBox | undefined;
   public scratchOrgAliasName: string | undefined;
   public scratchOrgId: string | undefined;
 
   public constructor(testSuiteSuffixName: string) {
     this.testSuiteSuffixName = testSuiteSuffixName;
+    this.tempFolderPath = path.join(__dirname, '..', 'e2e-temp');
```

Comment:
tempFolderPath should be initialized in constructor. The previous impl made code more complicated.